user/server-grpc-example.md

```diff
@@ -144,6 +145,11 @@ func main() {
 	if err != nil {
 		os.Exit(1)
 	}
+
+	if resp.Status != protocol.Ok {
```

Comment:
Since this example is for illustrative purposes, it may be interesting to add an error message per error to better explain what the problem is. In this case something like
_Source code has not been parsed correctly_, or similar.